moped-database/metadata/tables.yaml

```diff
@@ -153,6 +153,53 @@
     - role: moped-editor
       permission:
         filter: {}
+  event_triggers:
+    - name: activity_log_feature_drawn_lines
+      definition:
+        delete:
+          columns: '*'
+        enable_manual: false
+        insert:
+          columns: '*'
+        update:
+          columns: '*'
+      retry_conf:
+        interval_sec: 10
+        num_retries: 0
+        timeout_sec: 60
+      webhook_from_env: HASURA_ENDPOINT
+      headers:
+        - name: x-hasura-admin-secret
+          value_from_env: ACTIVITY_LOG_API_SECRET
+      request_transform:
+        body:
+          action: transform
+          template: |-
+            {
+              "query": "mutation InsertActivity($object: moped_activity_log_insert_input!) { insert_moped_activity_log_one(object: $object) { activity_id } }",
+              "variables": {
+                "object": {
+                  "record_id": {{ $body.event.data.new.id }},
+                  "record_type":  {{ $body.table.name }},
+                  "activity_id": {{ $body.id }},
+                  "record_data": {"event": {{ $body.event }}},
+                  "description": [{"newSchema": "true"}],
+                  "operation_type": {{ $body.event.op }},
+                  "updated_by_user_id": {{ $session_variables?['x-hasura-user-db-id'] ?? 1}}
+                }
+              }
+            }
```

Comment:
Just pointing out that because these records do not have a `project_id` column, we unfortunately cannot update the project's modified date, nor can we set the `record_project_id` column in the activity log. 

we'll leave that challenge for another today—i'm just glad to be tracking changes to these records.

also, i noticed that the `moped_activity_log.record_project_id` column defaults to `0`, which i think should be changed to `null`. i'm going to open a separate issue for that b/c i believe we can also clean up a few other columns.

TLDR—looks good!

Reply:
Thanks for that context, y'all! I was going to ask about the missing value in the `record_project_id` column, but, now, I understand why. 🙏